src/content/tutorials/fleet-management/cluster-management/cluster-autoscaler/index.md

```diff
@@ -0,0 +1,121 @@
+---
+linkTitle: Cluster autoscaler
+title: Advanced cluster autoscaler configuration
+description: Here we describe how you can customize the configuration of the managed cluster autoscaler service in your workload clusters.
+weight: 90
+menu:
+  principal:
+    parent: tutorials-fleet-management-clusters
+    identifier: tutorials-fleet-management-clusters-cluster-autoscaler
```

Comment:
Please update from main. These menu identifiers need adjustment to latest changed.

Reply:
Ignore me, I was looking at the wrong section.